.ci/code_coverage.py

```diff
@@ -5,6 +5,7 @@
 import subprocess
 import sys
 
+GCC_VERSION = 11
```

Comment:
While this is slightly better than before, even better would be to pass this in from somewhere outside.
Can be done later, FIXME comment maybe?